src/lib/reasoners/ccx.ml

```diff
@@ -145,79 +146,89 @@ module Main : S = struct
         Util.MI.iter
           (fun _ x -> fprintf fmt "%a |-> ... (See Uf)@." X.print x) mp
       in
-      if get_debug_cc () then begin
-        fprintf fmt "I am in %s with the following facts@." msg;
-        fprintf fmt "---- Begin Facts -----------------------------------@.";
-        fprintf fmt "Equalities:@.%a" aux f.equas;
-        fprintf fmt "Disequalities:@.%a" aux f.diseqs;
-        fprintf fmt "Inequalities:@.%a" aux f.ineqs;
-        fprintf fmt "Touched:@.%a" aux2 f.touched;
-        fprintf fmt "---- End   Facts -----------------------------------@.@.";
-      end
+      print_dbg ~debug:(get_debug_cc ())
+        ~module_name:"Ccx" ~function_name:"facts"
+        "I am in %s with the following facts@,\
+         ---- Begin Facts -----------------------------------@,\
+         Equalities:@,%a@,\
+         Disequalities:@,%a@,\
+         Inequalities:@,%a@,\
+         Touched:@,%a@,\
+         ---- End   Facts -----------------------------------@."
```

Comment:
We can't assume that we're in a vertical box here and this is clearly supposed to be in one:

```suggestion
        "@[<v 0>I am in %s with the following facts@,\
         ---- Begin Facts -----------------------------------@,\
         Equalities:@,%a@,\
         Disequalities:@,%a@,\
         Inequalities:@,%a@,\
         Touched:@,%a@,\
         ---- End   Facts -----------------------------------@]@."
```